example/helm/values.mysql.yaml

```diff
@@ -4,3 +4,25 @@ auth:
   existingSecret: db-credentials
 commonLabels:
   sidecar.istio.io/inject: "false"
+
+initdbScripts:
+  create_katib_database.sh: |
```

Comment:
this script will be executed on all nodes

`These scripts are treated differently depending on their extension. While .sh scripts are executed on all the nodes, .sql and .sql.gz scripts are only executed on the primary nodes. This is because .sh scripts support conditional tests to identify the type of node they are running on, while such tests are not supported in .sql or sql.gz files.
`
https://github.com/bitnami/charts/tree/main/bitnami/mysql#customize-a-new-mysql-instance
We either must add condition to check if it's primary node or use sql extension

Reply:
This makes sense. Please add this condition.